pkg/cmd/audit/audit.go

```diff
@@ -239,6 +255,20 @@ func (o *AuditOptions) Run() error {
 		filters = append(filters, &FilterByDuration{d})
 	}
 
+	if psaFilter := o.podsecurityfilter; len(psaFilter) > 0 {
```

Comment:
keep the pattern of the others: single filter type that encompasses all of the filter conditions instead of attaching an extra here.